### PR TITLE
a new layer for chapter titles

### PR DESCRIPTION
**Pull request by bijoshtj**

### CODE REVIEW

File: src/views/tiles/ChapterView.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		this._titleSettings = this._characterSettings.title;
+		this._idSettings = this._characterSettings.id;
+
+		this._titleText = new TextView(merge({
+			superview: this,
+		}, this._titleSettings));
+
+		this._idText = new ScoreView(merge({
+			superview: this
+		}, this._idSettings));
```
Comment (rampr) [inline src/views/tiles/ChapterView.js:22]: why do we have a textview and a score view ?

**Comment by rampr** [comment]
looks good to me otherwise

File: src/views/tiles/ChapterView.js (hunk 1 above)
Comment (bijoshtj) [reply src/views/tiles/ChapterView.js:22]: score view to show chapter nos, text view for chapter title can be localized as well.

**Review by vivekb91 — APPROVED** [review]

File: src/views/tiles/ChapterView.js (hunk 1 above)
Comment (rampr) [reply src/views/tiles/ChapterView.js:22]: okay understood

**Review by rampr — APPROVED** [review]

**Review by rampr — APPROVED** [review]

**Pull request merged**